localads/localads_tests/campaign_serialization_test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "testing/testing.hpp"
+
+#include "localads/campaign_serialization.hpp"
+
+#include <random>
+#include <vector>
+
+
+using namespace localads;
+
+namespace
+{
+bool TestSerialization(std::vector<Campaign> const & cs)
+{
+  auto const bytes = Serialize(cs);
+  LOG(LINFO, ("Src msg size:", (8+2+1) * cs.size(),
```

Comment:
What this magic constant is? (8 + 2 + 1) ? Clang format please, use sizeof(Campaign) to estimate the original vector size in bytes.